redis/src/cluster_slotmap.rs

```diff
@@ -32,8 +32,9 @@ pub(crate) enum ReadFromReplicaStrategy {
 
 #[derive(Debug, Default)]
 pub(crate) struct SlotMap {
-    slots: BTreeMap<u16, SlotMapValue>,
+    pub(crate) slots: BTreeMap<u16, SlotMapValue>,
```

Comment:
nit/performance: do we care about order here? i.e. why do we choose to use `BtreeMap` vs `HashMap`?
Also, when the key is a number and is well distributed, it is better to use `NoHasher` (https://crates.io/crates/nohash-hasher)

Reply:
@eifrah-aws since the BTree is already in use, I recommend that you keep using it - if you think there's a more performant alternative, see if you can measure a difference in a benchmark, and change this in a separate PR. My guess is that the difference would be completely negligible, assuming NoHasher can implement the relevant behavior requirements that guided the choice for BTree here.

Reply:
Out of the scope of the PR. can be done if needed in separate PR.